.gitignore

```diff
@@ -23,6 +23,7 @@
 /data/sql/custom/*
 /src/server/scripts/Custom/*
 !/src/server/scripts/Custom/README.md
+*.idx
 
```

Comment:
I'm sorry about this, i'm using clangd in vscode and it generates temporal idx files for indexing, I will revert this line change once pr state will get changed from draft pull request to normal pull request